mflowgen/common/fc-netlist-fixing/outputs/netlist-fixing.tcl

```diff
@@ -22,6 +25,61 @@ globalNetConnect CVSS -netlistOverride -pin VSSPST2 -singleInstance IOPAD_top_cu
 globalNetConnect VSS -netlistOverride -pin VSSPST1 -singleInstance IOPAD_top_cut_3
 globalNetConnect AVSS -netlistOverride -pin VSSPST2 -singleInstance IOPAD_top_cut_3
 
+#dragonphy power nets
+globalNetConnect AVDD -type pgpin -pin AVDD -inst iphy -override
+globalNetConnect AVSS -type pgpin -pin AVSS -inst iphy -override
+globalNetConnect CVDD -type pgpin -pin CVDD -inst iphy -override
+globalNetConnect CVSS -type pgpin -pin CVSS -inst iphy -override
```

Comment:
These are redundant with identical lines in globalnetconnect.tcl, yes? But I suppose that's okay.